src/Models/ElementalArea.php

```diff
@@ -112,7 +112,9 @@ public function setElementsCached(ArrayList $elements)
      */
     public function setOwnerPageCached(DataObject $page)
     {
-        $this->cacheData['owner_page'] = $page;
+        $cacheKey = 'owner_page_'. Versioned::get_reading_mode();
+
+        $this->cacheData[$cacheKey] = $page;
```

Comment:
```suggestion
        $this->cacheData['owner_page_' . Versioned::get_reading_mode()] = $page;
```

Reply:
I would leave it as it was for consistency with the changes below where it uses the variable in the same format.

Reply:
The only reason it's assigned as a variable below is because the key is used for a conditional check before being used in the array. That said this is such a small thing, I'm happy to be overruled in this case.